toolkit/src/components/cut/list-item/service-instance/index.hbs

```diff
@@ -0,0 +1,75 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+}}
+
+<Cut::ListItem
+  @href={{@href}}
+  @isRouteExternal={{@isRouteExternal}}
+  @route={{@route}}
+  @isHrefExternal={{@isHrefExternal}}
+  @query={{@query}}
+  @replace={{@replace}}
+  @onClick={{@onClick}}
+  data-test-service-instance-list-item
+  as |L|>
+  <L.Content as |C|>
+    <Cut::ListItem::Template as |T|>
+      <T.Label as |LB|>
+        <LB.Title data-test-service-name>{{@service.name}}</LB.Title>
+        <LB.Metadata>
+          {{#if this.isAllHealthy}}
+            <Hds::Badge
+              @color='success'
+              @icon='check'
+              @text='Healthy'
+              @size="small"
+              data-test-health-check-all-healthy
+            />
+          {{else}}
+            <!--Service health check-->
+            {{#if @service.metadata.healthCheck.service}}
+              <Cut::Metadata::HealthCheckBadgeSet
+                @successCount={{@service.metadata.healthCheck.service.success}}
+                @warningCount={{@service.metadata.healthCheck.service.warning}}
+                @criticalCount={{@service.metadata.healthCheck.service.critical}}
+                @type="service"
+              />
+            {{/if}}
+            <!--Node health check-->
+            {{#if @service.metadata.healthCheck.node}}
+              <Cut::Metadata::HealthCheckBadgeSet
+                @successCount={{@service.metadata.healthCheck.node.success}}
+                @warningCount={{@service.metadata.healthCheck.node.warning}}
+                @criticalCount={{@service.metadata.healthCheck.node.critical}}
+                @type="node"
+              />
+            {{/if}}
+          {{/if}}
+
+          <Cut::Metadata::InServiceMesh
+            @connectedWithProxy={{@service.metadata.connectedWithProxy}}
+          />
+
+          {{#if @service.metadata.externalSource}}
+            <Cut::Metadata::ExternalSource @externalSource={{@service.metadata.externalSource}}/>
```

Comment:
I wasn't seeing the external source on (at least) lambda in the top level service list on your consul draft. I was seeing it on the list items.

<img width="3120" alt="Screenshot 2023-06-06 at 10 37 05 AM" src="https://github.com/hashicorp/consul-ui-toolkit/assets/5448834/df9f79c8-b869-49f5-bd25-b49c615d2bb8">

Reply:
yea, it was removed intentionally, cause it is not a part of 'regular' service, from what I get. 

But I'll doublecheck with Jake, maybe it is just missed in design

Reply:
Updated list item type='service' to support externalSource and in service mash